package.json

```diff
@@ -62,6 +62,7 @@
     "publish:cd": "shared/deployments/publish-cd.sh",
     "publish:apps": "lerna run publish",
     "publish:http": "lerna run --scope='@tps/apps-*' publish:http --stream",
+    "recoverTemplate": "cd templates/open-enterprise && npm run recover:tx",
```

Comment:
Did you consider putting this script into a javascript file and adding it to a `bin` section here?
This way the potential users coming installing our packages from `npm` would have an easier way to call this, as a command, for example, the same as `aragon` is command runs for `@aragon/cli`...

I think for this use case might be more appropriate, since lot of users would not know the inner-workings of running `package.json` scripts or figure out if they should use `npm`, `yarn` or whatever... and it doesn't take a lot of effort to make that change...

Reply:
I saw this as something we would use to help support users who run into this issue. I don't think prompting users to install this repo is that the best way to go. If we want to give users a self-service option, building a a microsite on top of this logic that creates the tx for them in metamask would be a huge improvement for relatively little work, but we're not sure it's necessary yet.

Reply:
It seems a good idea Kevin, but right now, the way it is implemented there is no chance to run this without installing all the dependencies, like truffle, @aragon/cli, node, etc, so I in its current state I think it is a better idea we think of it as a tool for us (Autark) to provide support to others, but it is still a bit complex to use for a "regular" user in those terms...